reports/pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[project]
```

Comment:
quando vc deu o init ele nao alterou o pyproject.toml da raiz para adicionar o workspace? to sentindo falta desta mudança no diff, exemplo:

```diff
Modified regular file pyproject.toml:
    ...
  57   57:     "regular: regular time expected execution magnitude",
  58   58:     "slow: slow expected execution magnitude",
  59   59: ]
       60: 
       61: [tool.uv.workspace]
       62: members = ["reports"]
```